app/uk/gov/hmrc/apiplatform/modules/mfa/connectors/ThirdPartyDeveloperMfaConnector.scala

```diff
@@ -77,16 +77,14 @@ class ThirdPartyDeveloperMfaConnector @Inject()(http: HttpClient, config: Applic
     }
   }
 
-  def removeMfa(userId: UserId, email: String)(implicit hc: HeaderCarrier): Future[Unit] = {
-    implicit val RemoveMfaRequestFormat = Json.format[RemoveMfaRequest]
+  def removeMfaById(userId: UserId, mfaId: MfaId, email: String)(implicit hc: HeaderCarrier): Future[Unit] = {
+    implicit val removeMfaByIdRequestFormat = Json.format[RemoveMfaByIdRequest]
```

Comment:
Put this in the companion of the case class (you can nest object within an object)

Reply:
Done